docs/tutorial.rst

```diff
@@ -19,6 +19,9 @@ The most basic configuration is just telling ``towncrier`` where to look for new
 
    [tool.towncrier]
    directory = "changes"
+   # Where you want your news files to come out. This can be .rst
+   # or .md, towncrier's default template works with both.
+   filename = "NEWS.rst"
```

Comment:
> The most basic configuration is just telling ``towncrier`` where to look for news fragments::

Now we also need to specify the filename ?

```diff
-The most basic configuration is just telling ``towncrier`` where to look for news fragments::
+The most basic configuration is just telling ``towncrier`` where to look for news fragments, and it will generate a `NEWS.rst` file in the project root::
```

Reply:
I now changed it to

> The most basic configuration is just telling ``towncrier`` where to look for news fragments and what file to generate::

to keep the same style of only saying what it does, without giving the value of the option (which comes in the later paragraph).